packages/core/app.plugin.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./plugin/build');
```

Comment:
I'm curious why this file is needed in the `core` package. So that it is visible by the Expo, I guess?

Reply:
that's some leftover from a previous try, I'll remove that!